addons/medical/functions/fnc_isMedic.sqf

```diff
@@ -19,7 +19,7 @@
 
 params ["_unit", ["_medicN", 1]];
 
-private _class = _unit getVariable [QGVAR(medicClass), getNumber (configFile >> "CfgVehicles" >> typeOf _unit >> "attendant")];
+private _class = _unit getVariable [QGVAR(medicClass), _unit getUnitTrait "medic"];
```

Comment:
_class should be number, not bool

someone in public slack was talking about using config `attendant =2` to set doctors
it's not the prettiest code, but something like this should support that
```
private _class = _unit getVariable QGVAR(medicClass);
if (isNil "_class") then {
    if (_unit getUnitTrait "medic") then {
        _class = 1 max getNumber (configFile >> "CfgVehicles" >> typeOf _unit >> "attendant");
    } else{
        _class = 0;
    };
};
```